### PR TITLE
Add new dataset sa-edu lecture 1.2

### PR DESCRIPTION
**Pull request by delapraznik**

### CODE REVIEW

File: core/sa-edu-dental-fillings.tab.info [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        "survival analysis",
+        "censoring"
+    ],
+    "target": "",
+    "version": "1.2",
+    "year": 2022,
+    "instances": 20,
+    "missing": false,
+    "variables": 4,
+    "source": "",
```
Comment (lanzagar) [inline core/sa-edu-dental-fillings.tab.info:16]: If you did not generate the file yourself, try to put the link to where the data came from in source or a paper/website citation as a reference.

Comment (delapraznik) [reply core/sa-edu-dental-fillings.tab.info:16]: I generated it myself.

Comment (lanzagar) [reply core/sa-edu-dental-fillings.tab.info:16]: In that case I suggest you leave this blank, and after you publish a blog using this data you edit this info and add a link to the blog to "seealso" :)

**Comment by JakaKokosar** [comment]
LGTM, @lanzagar can we merge this?

**Pull request merged**